vendor/composer/autoload_classmap.php

```diff
@@ -3741,6 +3749,7 @@
     'services\\ChartTrackerService' => $baseDir . '/services/ChartTrackerService.php',
     'services\\FacilityService' => $baseDir . '/services/FacilityService.php',
     'services\\ONoteService' => $baseDir . '/services/ONoteService.php',
+    'services\\PatientService' => $baseDir . '/services/PatientService.php',
```

Comment:
Yep, it was generated when I ran `composer dump-autoload -o`. Apparently that file made its way into the master branch w/o getting the autoloader redone.